app/Models/Location.php

```diff
@@ -17,6 +17,7 @@ class Location extends AbstractModel
         'page_id',
         'landing_page_id',
         'name',
+        'directions'
```

Comment:
Maybe this is pedantic, but I think this attribute should be named `directions_link` because `directions` makes me think it's the actual text of the directions, eg "Turn left at the green lions". Is that too nit-picky?

Reply:
I think this is a valid concern and it doesn't hurt at all to be more specific in this case.